demo/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "checkJs": true,
+    "module": "node16",
+    "noEmit": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "paths": {
+      "@mdx-js/monaco": ["../packages/monaco/index.js"]
```

Comment:
Could/should this be solved by using a npm/yarn workspace?

Reply:
The `paths` option lets us use the types without building it first. I figured this out, because it annoyed me that after every edit to a dependency I had to compile to fix the type errors in the dependant package.

In fact, this can be applied to other projects in the unified ecosystem as well.

For example [remark workspaces](https://github.com/remarkjs/remark/blob/main/package.json#L13-L18) don’t need to be sorted. It could just way `"workspaces": ["packages/*"]`.